config/routes.rb

```diff
@@ -2,4 +2,6 @@
 
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  scope '/:locale' do
+  end
```

Comment:
I think we can make do without this scope in the routes. The applicationController will pick up the locale from the query params even if you haven't specified that in the routes file